Dizzy/Screens/SignIn/SignInDetailsView.swift

```diff
@@ -59,7 +59,7 @@ final class SignInDetailsView: UIView {
     private func layoutStackView() {
         self.addSubviews([self.stackView])
         self.stackView.snp.makeConstraints { stackView in
-            stackView.top.equalTo(titleLabel.snp.bottom).offset(Metrics.doublePadding)
+            stackView.top.equalTo(titleLabel.snp.bottom).offset(Metrics.doublePadding * 2)
```

Comment:
don't use `*2`, you can create a constant like `stackViewTopPadding = ..` or create in Metrics new variable and use it